scripts/proxy.api.sh

```diff
@@ -42,7 +42,11 @@ gitRepo()
       rm -rf $DIR/temp
       rm -rf $DIR_REPO/.git
 
-      echo $(date -v +10d "+%s") > $DIR/expire.txt
+      if [ "$(uname)" = "Darwin" ]; then
```

Comment:
You'll probably want a similar check on [line 170](https://github.com/RedHatInsights/curiosity-frontend/pull/340/files#diff-6fa35ea8fe8d4e6040981359e9a7af4bR170) to use `xdg-open` instead of `open` when on linux.